frontend/src/components/MapFiltersMenu/MapFiltersMenu.tsx

```diff
@@ -85,8 +85,6 @@ const MapFiltersMenu = ({
     };
 
     const resetFilters = () => {
-        mapFilters.publicSince = "";
-        mapFilters.publicUntil = "";
         setMapFilters(initFilters);
         updateFilters({ ...initFilters });
     };
```

Comment:
Clicking "Tyhjennä" (Reset) does empty the filters, but text input content does not disappear. It does so with "name contains" filter text input, but here in "published hidden" filter it does not happen. Well, it happens, if you update the map by clicking "Etsi alueelta" (search in the visible area), but does not by using "Tyhjennä" (Reset).